CMakeLists.txt

```diff
@@ -187,6 +204,7 @@ set(AVIF_SRCS
     src/read.c
     src/reformat.c
     src/reformat_libyuv.c
+    src/reformat_libsharpyuv.c
```

Comment:
Nit: The files here are listed in alphabetical order, so this should be listed before `src/reformat_libyuv.c`.